drive/motor.h

```diff
@@ -0,0 +1,25 @@
+#ifndef motor_h
+  #define motor_h
+  #pragma once
+  
+  class motor {
+    private:
+      int motorSpeed;
+      int pin1;
+      int pin2;
```

Comment:
Reformat variable names following the [convention](https://google.github.io/styleguide/cppguide.html#Naming):
```c++
int pin_speed_;
int pin_1_;
```